app/waf/src/services/trustedproxy.service.ts

```diff
@@ -0,0 +1,55 @@
+import {getService} from '@loopback/service-proxy';
```

Comment:
Maybe we can merge it with asg.service.ts

Reply:
There are 2 reasons I didn't do that:
1. asg* staff is frequently coding changing. Coding it within that file will lead more merging effort. 
2. We can use asg as a datasource name, but the kinds of functionalities are provided by extensions. 
What functions asg provides depends on what extensions it has. I think separate the extensions into separate files can make us easier to decouple changes.

Reply:
Moved to ASGManager.